packages/bpk-component-chip-group/src/BpkChipGroup.tsx

```diff
@@ -0,0 +1,195 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { ReactElement, ReactNode} from 'react';
+import { useRef, useState } from 'react';
+
+import { cssModules, isRTL } from '../../bpk-react-utils';
+import BpkSelectableChip, { type BpkSelectableChipProps, CHIP_TYPES } from '../../bpk-component-chip';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import BpkMobileScrollContainer from '../../bpk-component-mobile-scroll-container';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import FilterIconSm from '../../bpk-component-icon/sm/filter';
+import BpkBreakpoint, { BREAKPOINTS } from '../../bpk-component-breakpoint';
+
+import Nudger from './Nudger';
+import STYLES from './BpkChipGroup.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+export const CHIP_GROUP_TYPES = {
+  rail: 'rail',
+  wrap: 'wrap',
+}
+export type ChipGroupType = (typeof CHIP_GROUP_TYPES)[keyof typeof CHIP_GROUP_TYPES];
+export type ChipStyleType = (typeof CHIP_TYPES)[keyof typeof CHIP_TYPES];
+
+export type SingleSelectChipItem = {
+  text: string;
+  accessibilityLabel?: string;
+  leadingAccessoryView?: ReactNode;
+  className?: string;
+  [rest: string]: any; // Inexact rest. See decisions/inexact-rest.md
+};
+
+export type ChipItem = {
+  component?: (props: BpkSelectableChipProps) => ReactElement;
+  onClick?: (selected: boolean, index: number) => void;
+  selected?: boolean;
+} & SingleSelectChipItem;
```

Comment:
The shape of this is a bit different to the figma at the moment but I've tried to keep it as close as possible: https://www.figma.com/file/irZ3YBx8vOm16ICkAr7mB3/Backpack?node-id=30119%3A42328&mode=dev

Currently differs by:
- `icon` replaced by `leadingAccessoryView` as this is the prop in BpkSelectableChipProps, although this could be renamed to icon for this prop if preferred.
- `component` instead of `type` but could be made to use an enum to choose from `selectable`, `dropdown` and `dismissable` chips.

Since these just passed through to a BpkChip component then a `...rest` param seemed sensible at the time, although may not be neccessary now I'm looking back.